next-sitemap.js

```diff
@@ -72,6 +86,10 @@ module.exports = {
   },
   additionalPaths: async (config) => {
     const result = [];
+    let tafsirIDs = [];
+    await getTafsirsIds().then((response) => {
+      tafsirIDs = response.tafsirs.map((tafsir) => tafsir.id);
```

Comment:
we can use slugs instead of tafsir id

https://staging.quran.com/api/qdc/resources/tafsirs

`en-tafisr-ibn-kathir` or 169?

Reply:
"Get tafsir api" support both ids and slugs 

https://staging.quran.com/api/qdc/tafsirs/en-tafisr-ibn-kathir/by_ayah/1:2?words=true&fields=text_uthmani

Reply:
@naveed-ahmad JazakAllah Kahiran, definitely the slugs are better but need to handle them first in the route. Thanks for letting me know that we can use the slugs!